File: in_Java/1051 Height Checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int[] x = new int[heights.length];
        int res = 0;
        for(int i = 0; i < x.length; i++) x[i] = heights[i];
        Arrays.sort(x);
        for(int i = 0; i < x.length; i++){
            if(x[i] != heights[i]) res++;
        }
        return res;
    }
}

File: in_Python/1051 Height Checker.py
class Solution(object):
    def heightChecker(self, heights):
        """
        :type heights: List[int]
        :rtype: int
        """
        h = []
        for hh in heights:
            h.append(hh)
        h.sort()
        res = 0
        for i in range(len(h)):
            if h[i] != heights[i]:
                res += 1
        return res